documentation/metadata.md

```diff
@@ -25,591 +24,1100 @@ This page document the exact format that the JSON files containing PyDough metad
+> [!IMPORTANT]
+> This type of metadata relationship has not yet been implemented.
 
-- `primary_property`: a string indicating the name of a property that connects this collection to a subcollection.
-- `secondary_property`: a string indicating the name of the property that connects the primary subcollection to one of its subcollections.
-- `singular`: a boolean that is true if each record in the current collection has at most 1 matching record of the subcollection’s subcollection, and false otherwise.
-- `no_collisions`: a boolean that is true if multiple records from this collection can match onto the same record from the subcollection’s subcollection, and false otherwise (true if-and-only-if the reverse relationship is singular).
-- `reverse_relationship_name`: the name of the property that is to be added to the other collection to describe the reverse version of this relationship. This string must be a valid property name but cannot be equal to another existing property name in the other collection.
-- `inherited_properties`: a JSON object indicating any properties of the primary subcollection that should be accessible from the secondary subcollection. The keys are the string names that the inherited properties are referred to by, which can be a new alias or the original name, and the values are names of the properties of the collection accessed by the primary property. The names used for the inherited properties cannot overlap with any other names of properties of the secondary subcollection, including other inherited properties it could have from other compound relationships. This JSON object can be empty if there are no inherited properties.
+<!-- TOC --><a name="relationship-type-reverse"></a>
+### Relationship Type: Reverse
 
-Example of the structure of the metadata for a compound relationship property (connects a collection `Regions` to a collection `Customers` by combining `Regions.nations.customers` into a single property, skipping over a middle collection `Nations` but picking up its property `name` under a new name `nation_name):
+A relationship with this type specifies that a relationship should be created by flipping the direction of an existing relationship that was already defined earlier in the `relationships` array. Relationships of this type are sub-collections of the collection, as opposed to scalar attributes. Relationships of this type have a type string of "cartesian product" and have the following additional key-value pairs in their metadata JSON object:
+- `original parent` (required): a string indicating the name of the collection who holds the relationship being reversed (this will be the child of hte new property)
```

Comment:
```suggestion
- `original parent` (required): a string indicating the name of the collection who holds the relationship being reversed (this will be the child of the new property)
```